jwst/firstframe/firstframe_step.py

```diff
@@ -0,0 +1,29 @@
+from ..stpipe import Step
+from .. import datamodels
+from . import firstframe_sub
+
+
+class FirstFrameStep(Step):
+    """
+    FirstFrameStep: This is a MIRI specific task.  If the number of groups
+    is greater than 1, the GROUP data quality flags for the first group will
+    be set to DO_NOT_USE.
```

Comment:
Again, let's phrase it as "added to", rather than "set to".

Reply:
Done.